src/bin/flamegraph.rs

```diff
@@ -51,16 +98,18 @@ impl Into<Options> for Opt {
             },
             None => FuncFrameAttrsMap::default(),
         };
+
         let direction = if self.inverted {
             Direction::Inverted
         } else {
             Direction::Straight
         };
-        let title = if self.inverted {
-            "Icicle Graph".to_string()
-        } else {
-            "Flame Graph".to_string()
+
+        let title = match self.title.as_str() {
+            "Flame Graph" if self.inverted => "Icicle Graph".to_string(),
```

Comment:
Why would it ever be set to the string "Flame Graph"? That hasn't been declared anywhere else as far as I can see? Ideally I'd like it if the two "default" titles were both defined in the same place.

Reply:
missed a default earlier